libcasr/fuzz/fuzz_targets/parse_stacktrace.rs

```diff
@@ -0,0 +1,49 @@
+#![no_main]
+
+use libfuzzer_sys::fuzz_target;
+
+use libcasr::asan::AsanStacktrace;
+use libcasr::gdb::GdbStacktrace;
+use libcasr::go::GoStacktrace;
+use libcasr::java::JavaStacktrace;
+use libcasr::python::PythonStacktrace;
+use libcasr::stacktrace::ParseStacktrace;
+
+fuzz_target!(|data: &[u8]| {
+    if data.len() < 2 {
+        return;
+    }
+    let s = String::from_utf8_lossy(&data[1..]);
+    match data[0] % 5 {
```

Comment:
May be we should split `parse_stacktrace.rs` to several separate ones for each `ParseStacktrace` impl?

Reply:
The implementations of parsers are small. I think we could try this way first. If coverage is bad, we could separate them.